File: python/tasks/merge_two_arrays.py
arr = [1, 2, 3, 4]
arr2 = [5, 6, 7, 8, 9, 10]
arr3 = []

for i in range(0, len(arr)-1):
    arr3.append(arr[i])


for i in range(0, len(arr2)):
    arr3.append(arr2[i])


print(arr3)

File: src/Java/tasks/MergeTwoArrays.java
package Java.tasks;

import java.util.Arrays;

public class MergeTwoArrays {
    public static void main(String[] args) {

        int[] arr = {1,2,3,4,5};
        int[] arr2 = {6,7,8,9,10};
        int[] arr3 =new int[arr.length+arr2.length];
        int j = 0;
        for (int each : arr){
            arr3[j++] = each;
        }

        for (int each : arr2){
            arr3[j++] = each;
        }

        System.out.println(Arrays.toString(arr3));
    }
}
